src/engine/source/api/src/kvdb/handlers.cpp

```diff
@@ -104,19 +117,32 @@ api::Handler managerDelete(std::shared_ptr<kvdb_manager::KVDBManager> kvdbManage
             return ::api::adapter::genericError<ResponseType>("Missing /name");
         }
 
-        // Adapt the response to wazuh api
+        if (eRequest.name() == "")
+        {
+            return ::api::adapter::genericError<ResponseType>("Database name is empty");
+        }
+
+        auto exists = kvdbManager->existsDB(eRequest.name());
+
+        if (exists.has_value())
```

Comment:
If `exists` is used just here, maybe the method could be directly used instead of using a variable

Reply:
The same goes for all the occurrences

Reply:
The exists is using in the line 129: exists.value().